admin_manual/release_notes.rst

```diff
@@ -16,6 +17,102 @@ Release Notes
+For security reasons the default value for minimum characters to trigger the sharing autocomplete mechanism has been set to "4" (previously it was set to "2"). This is to impede user name or mail address tapping. As it is a trade-off between security and usability for some scenarios this high security level might not be desirable. Therefore the value now is configurable via the *config.php* option ``'user.search_min_length' => 4,``. Please check which value fits your needs best.
+
+New option to granularly configure public link password enforcement
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+With ownCloud 10 the *"File Drop"* feature has been merged to public link permissions. As this kind of public link does not give recipients access to any contents but just gives them the possibility to "drop" files it might not be desirable for some scenarios to enforce password protection for such shares while other public links should require setting a password. For this password enforcement for public links has been split to apply based on permissions given (*read-only, read & write, upload only/File Drop*). Please check the administration settings *'Sharing'* section and configure as desired.
+
+New option to exclude apps from integrity check
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+By verifying signature files the *integrity check* ensures that the code running in an ownCloud instance has not been altered by third parties. Naturally this check can only be successful for code that has been obtained from official ownCloud sources. In the case of external modifications (e.g., for theming) the *integrity check* will fail and notify the administrator. For intended modifications within apps these can now be excluded from the *integrity check* by using the *config.php* option ``'integrity.ignore.missing.app.signature' => ['app1', 'app2', 'app3'],``. See *config.sample.php* for more information.
+
+New occ command to modify user details
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+It is now possible to modify user details like display names or mail addresses via the command ``occ user:modify``. Please append ``--help`` for more information.
+
+occ files:scan can now be executed for groups
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Apart from using the ``occ files:scan`` command for *single users* and *whole instances* it can now be executed for *groups* using ``-g, --groups=GROUPS``. Please append ``--help`` for more information.
+
+New configurable default format for syslog
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When using syslog as the log type (``'log_type' => 'syslog',`` in *config.php*) the default format has been changed to include *request IDs* for easier debugging. Additionally the log format has been made configurable using ``'log.syslog.format'`` in *config.php*. If you require a certain log format to forward to external applications, please check the new format and *config.sample.php* on how to change it.
```

Comment:
I think you don't forward it to external applications as it's about syslog. So you just specify what format to use for syslog and the messages are forwarded to syslog, not external applications.

Reply:
I thought about fail2ban or something, nvm